third_party/xla_client/BUILD

```diff
@@ -485,8 +479,8 @@ tf_cc_shared_object(
         "@org_tensorflow//tensorflow/compiler/xla/client/lib:svd",
         "@org_tensorflow//tensorflow/compiler/xla/rpc:grpc_stub",
         "@org_tensorflow//tensorflow/core:lib",
```

Comment:
same question, do we need this `tensorflow/core` deps directly or generally?

we would migrate to pull XLA from OpenXLA, and would delete tensorflow too, so would delete `tensorflow/core` deps too